AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -941,11 +1017,14 @@ - (void)testMigrationFromSchema2to3 {
     [logs addObject:log];
   }
 
-  // Check the insertion worked.
-  trueLogCount = [storage countEntriesForTable:kMSLogTableName
-                                     condition:[NSString stringWithFormat:@"\"%@\" = '%@' AND \"%@\" = %u", kMSGroupIdColumnName, groupId,
-                                                                          kMSPriorityColumnName, (unsigned int)flags]];
-  assertThatUnsignedInteger(trueLogCount, equalToUnsignedInteger(count));
+  if (verify) {
+
+    // Check the insertion worked.
+    trueLogCount = [storage countEntriesForTable:kMSLogTableName
```

Comment:
What is a logCount that is not true? A non working log count? Not sure what is the intent here.